src/System.Console/src/System/ConsolePal.Unix.cs

```diff
@@ -74,7 +74,7 @@ internal static TextReader GetOrCreateReader()
                     StreamReader.Null :
                     new StreamReader(
                         stream: inputStream,
-                        encoding: new ConsoleEncoding(Console.InputEncoding), // This ensures no prefix is written to the stream.
+                        encoding: Console.InputEncoding.RemovePreamble(), // This ensures no prefix is written to the stream.
```

Comment:
I don't understand these changes.  ConsoleEncoding's GetPreamble always returns an empty array, so what benefit is RemovePreamble providing here?

Reply:
This one is a no-op. I'll remove it.